### PR TITLE
Clarify that fog_aerial_perspective does not actually use sky color

### PR DESCRIPTION
**Pull request by ttencate**
See #97803.

### CODE REVIEW

**Review by Calinou — APPROVED** [review]

**Review by fire — APPROVED** [review]

File: doc/classes/Environment.xml [hunk 1]
```diff
@@ -81,8 +81,9 @@
...
-			If set above [code]0.0[/code] (exclusive), blends between the fog's color and the color of the background [Sky]. This has a small performance cost when set above [code]0.0[/code]. Must have [member background_mode] set to [constant BG_SKY].
+			If set above [code]0.0[/code] (exclusive), blends between the fog's color and the color of the background [Sky], as read from the radiance cubemap. This has a small performance cost when set above [code]0.0[/code]. Must have [member background_mode] set to [constant BG_SKY].
 			This is useful to simulate [url=https://en.wikipedia.org/wiki/Aerial_perspective]aerial perspective[/url] in large scenes with low density fog. However, it is not very useful for high-density fog, as the sky will shine through. When set to [code]1.0[/code], the fog color comes completely from the [Sky]. If set to [code]0.0[/code], aerial perspective is disabled.
+			Notice that this does not sample the [Sky] directly, but rather the radiance cubemap, at a low resolution mipmap level. This results in the actual color being a heavily blurred version of the sky, not an exact copy.
```
Comment (clayjohn) [inline doc/classes/Environment.xml:86]: This is only true for objects close to the camera. The mip level increases resolution as you get closer to the far plane.

Comment (ttencate) [reply doc/classes/Environment.xml]: Clarified to explain what I think is actually happening, please take another look!

When I write it like this, it makes me wonder _why_ it's done in this way...

**Review by clayjohn — APPROVED** [review]
Looks great to me! Thank you!

File: doc/classes/Environment.xml (hunk 1 above)
Comment (clayjohn) [reply doc/classes/Environment.xml]: We use the technique from the Uncharted games (described here: https://advances.realtimerendering.com/other/2016/naughty_dog/NaughtyDog_TechArt_Final.pdf)

**Comment by clayjohn** [comment]
Thank you!

**Pull request merged**